vllm/sequence.py

```diff
@@ -371,15 +389,20 @@ def __init__(
         sampling_params: SamplingParams,
         block_tables: Dict[int, List[int]],
         lora_request: Optional[LoRARequest] = None,
-        prefix: Optional[Prefix] = None,
+        prefix_pos: Optional[int] = None,
+        computed_block_nums: Optional[List[int]] = None,
```

Comment:
Are these two numbers duplicated? Can we infer the `prefix_pos` via `computed_block_nums`?